1-estrategia/README.md

```diff
@@ -1,23 +1,162 @@
+
+**ENCUESTA: CONSUMO DE LA MÚSICA EN INTERNET**
+
+¡Hola! Queremos diseñar una página web musical para ofrecer información de todo tipo acerca de este tema. Para ello, estamos realizando una investigación sobre el consumo de la música en Internet y necesitamos tu ayuda. Por favor, rellena este formulario cuando puedas. Muchas gracias.
+
+¿En qué formato consumes información en Internet? 
+- Texto 
+- Audio 
+- Video 
+- Imagen 
+
+¿Que tipo de contenidos consumes acerca de la música? 
+- Noticia
+- Reportaje 
+- Entrevista 
+- Videoclip 
+- Audio
+- Crítica
+
+Valora del 0 al 5 el interés que tengas acerca de los siguientes géneros musicales: (0 no me interesa, 5 me interesa mucho)
+- Música clásica  
+- Rock 
+- Pop 
+- Reggaetón 
+- Jazz 
+- Trap 
+- Punk 
+- Rap 
+- Techno 
+- Flamenco 
+- Country 
+
+Si tienes interés en otro género musical escríbelo aquí:
+
+¿Tienes interés en saber cuales son las tendencias musicales de la actualidad? Sí/No
+
+¿Y las tendencias o nuevos temas del género/s que más escuchas? Sí/No
+
+¿Te interesa profundizar acerca de las canciones que escuchas? SÍ/No
+
+Si has respondido que sí, ¿en qué te interesa profundizar?
+- Autor  
+- Letra 
+- El por qué de la canción 
+- Significado 
+- Su repercusión 
+
+¿Cómo consigues información acerca de los conciertos/festivales que vaya a haber?
+- Internet
+- Boca a boca
+- Carteles publicitarios
+- Periódico 
+- Televisión
+- Radio
+
+¿Compras entradas por Internet? Sí/No
+
+Si has respondido que sí, ¿cómo lo haces?
+- Acudo directamente a webs de venta de entradas 
+- Escribo en mi navegador frases tipo: “entradas para concierto X”  
+- Acudo a la página web del artista/grupo  
+- Acudo a páginas web musicales que me informan sobre la venta de entradas 
+- Ninguna de las anteriores
+
+Una vez realizada la encuesta y analizadas las respuestas obtenidas, los resultados y las conclusiones que hemos sacado son las siguientes:
+
+**Tipo de formato y contenido para consumir información musical**
+
+Las primeras dos preguntas de la encuesta han sido sobre el tipo de formato y contenido que utilizan a la hora de consumir información sobre música. En el caso del tipo de formato, los resultados han sido muy parecidos. El porcentaje más alto (80% de los encuestados) y por tanto, el tipo de contenido que más consumen son el texto, el vídeo y la imagen. El audio ha obtenido un porcentaje bastante más bajo (20%), pero al obtener un porcentaje que no sea el 0, creemos que también podría ser una forma de insertar contenido en nuestra página web, aunque no se utilizará más que los demás formatos.
```

Comment:
¿Texto, video e imagen al mismo nivel? ¿Es eso lo que queréis decir cuando decís que _los resultados han sido muy parecidos_?

Es cierto que no es 0, pero la diferencia con los demás es muy grande, y el esfuerzo de producción puede no resultar rentable.